conductor/houdini/hda/frame_spec_ui.py

```diff
@@ -0,0 +1,203 @@
+"""frame range section in the UI."""
+
+import hou
+from conductor.houdini.hda import driver_ui, takes, uistate
+from conductor.houdini.lib.sequence.sequence import Sequence
+
+# Specify that Expressions are Python
+XP = hou.exprLanguage.Python
+
+
+def _replace_with_input_expression(parm, input_path):
+    """Write expression based on equivalent parm on input node."""
+    parm.deleteAllKeyframes()
+    parm.setExpression('ch("%s")' % input_path, XP, True)
+
+
+def _set_to_input(node, rop):
+    """Give all channels an expression to link the input node."""
+
+    if not rop:
+        return
+    rop_path = rop.path()
+    for parm in ['1', '2', '3']:
+        _replace_with_input_expression(
+            node.parm('fs%s' % parm), '%s/f%s' % (rop_path, parm))
+
+
+def _set_to_scene(node):
+    """Give all channels an expression to link the global settings."""
+    node.parm('fs1').setExpression(
+        'hou.playbar.timelineRange()[0]', XP, True)
+    node.parm('fs2').setExpression(
+        'hou.playbar.timelineRange()[1]', XP, True)
+    node.parm('fs3').setExpression(
+        'hou.playbar.frameIncrement()', XP, True)
+
+
+def custom_frame_sequence(node):
+    """Generate Sequence from value in custom_range parm."""
+    spec = node.parm("custom_range").eval()
+    clump_size = node.parm("clump_size").eval()
```

Comment:
The function has changed since - it now uses a try/except block. I guess the above suggestion still stands? This is what I did: 
``` try:
        spec = node.parm("custom_range").eval()
        chunk = _chunk_parameters(node)
        return Sequence.create(
            spec,
            chunk_size=chunk["size"],
            chunk_strategy=chunk["strategy"]
        )
    except (ValueError, TypeError):
        return None```